docs/ops/detection/GenerateProposals_9.md

```diff
@@ -0,0 +1,148 @@
+    * **Range of values**: non-negative floating-point number
+    * **Type**: float
+    * **Required**: *yes*
+
+* *nms_threshold*
+
+    * **Description**: The *nms_threshold* attribute specifies threshold to be used in the NMS stage.
+    * **Range of values**: non-negative floating-point number
+    * **Type**: float
+    * **Required**: *yes*
+
+* *pre_nms_count*
+
+    * **Description**: The *pre_nms_count* attribute specifies number of top-n proposals before NMS.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Required**: *yes*
+
+* *post_nms_count*
+
+    * **Description**: The *post_nms_count* attribute specifies number of top-n proposals after NMS.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Required**: *yes*
+
+* *normalized*
+
+    * **Description**: *normalized* is a flag that indicates whether proposal bboxes are normalized or not.
+    * **Range of values**: true or false
+      * *true* - the bbox coordinates are normalized.
+      * *false* - the bbox coordinates are not normalized.
+    * **Type**: boolean
+    * **Default value**: True
+    * **Required**: *no*
+
+* *nms_eta*
+
+    * **Description**: eta parameter for adaptive NMS.
+    * **Range of values**: a floating-point number in close range `[0, 1.0]`.
+    * **Type**: float
+    * **Default value**: `1.0`
+    * **Required**: *no*
+
+* *roi_num_type*
+
+    * **Description**: the type of element of output 3 `rpnroisnum`.
+    * **Range of values**: i32, i64
+    * **Type**: string
+    * **Default value**: `i64`
+    * **Required**: *no*
+
+**Inputs**
+
+* **1**: `im_info` - tensor of type *T* and shape `[num_batches, 3]` or `[num_batches, 4]` providing input image info. The image info is layout as `[image_height, image_width, scale_height_and_width]` or as `[image_height, image_width, scale_height, scale_width]`. **Required.**
+
+* **2**: `anchors` - tensor of type *T* with shape `[height, width, number_of_anchors, 4]` providing anchors. Each anchor is layouted as `[xmin, ymin, xmax, ymax]`. **Required.**
+
+* **3**: `boxesdeltas` - tensor of type *T* with shape `[num_batches, number_of_anchors * 4, height, width]` providing deltas for anchors. The delta consists of 4 element tuples with layout `[dx, dy, log(dw), log(dh)]`. **Required.**
+
+* **4**: `scores` - tensor of type *T* with shape `[num_batches, number_of_anchors, height, width]` providing proposals scores.  **Required.**
+
+The `height` and `width` from inputs `anchors`, `boxesdeltas` and `scores` are the height and width of feature maps.
+
+**Outputs**
+
+* **1**: `rpnrois` - tensor of type *T* with shape `[num_rois, 4]` providing proposed ROIs. The proposals are layouted as `[xmin, ymin, xmax, ymax]`. The `num_rois` means the total proposals number of all the images in one batch.
+
+* **2**: `rpnscores` - tensor of type *T* with shape `[num_rois, 1]` providing proposed ROIs scores.
```

Comment:
please check why it is not just of shape `[num_rois]`.

Reply:
[num_rois] is also ok, have updated.